nixos/modules/services/monitoring/prometheus/sachet.nix

```diff
@@ -0,0 +1,79 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.prometheus.sachet;
+  configFile = pkgs.writeText "sachet.yml" (builtins.toJSON cfg.configuration);
+in
+{
+  options = {
+    services.prometheus.sachet = {
+      enable = mkEnableOption "Sachet is an SMS alerting tool for the Prometheus Alertmanager";
+
+      configuration = mkOption {
+        type = types.nullOr types.attrs;
+        default = null;
+        example = literalExample ''
+          {
+            providers = {
+              twilio = {
+                account_sid = "aCb3bbaacc554b";
+                auth_token = "4736473aaabbcc66";
+              };
```

Comment:
I don't feel great about putting credentials into the nix store.

Reply:
Hey, I'd like to revisit this PR. Unfortunately the configuration file has no way to specify the use of any environment variable. As such I don't see how I can avoid doing this.

Reply:
Actually it does parse environment variables by default, so I've added that instead in the example!

Reply:
I've homebrewed a way with envsubst that seems to work.